src/helpers/getDeckAdvice/advice/OBSIDIAN_BUTCHERS/index.js

```diff
@@ -9,27 +11,38 @@ export default cards => {
   const hasObsidianButchers = cardIds.includes('F17')
   const hasRainOfFrogs = cardIds.includes('F8')
   const hasAzureHatchers = cardIds.includes('F10')
+  const hasLudicMatriarchs = cardIds.includes('N43')
+  let hasLowLvlMatriarchs = false
+  if (hasLudicMatriarchs) {
+    hasLowLvlMatriarchs = cards.find( card =>  card.id === 'N43' ).level <= 2 ? true : false
```

Comment:
These spaces within the parentheses do not respect the Prettier conventions and should have been automatically fixed for you when you committed the code via the pre-commit hook. I’m surprised it didn’t. Maybe some dependencies were missing locally?